pkg/clusteragent/autoscaling/workload/dump_test.go

```diff
@@ -175,7 +175,7 @@ Settings: map[key:value]
 
 func TestMarshalUnmarshal(t *testing.T) {
 	// json serialization drops nanoseconds; strip it here
-	testTime := time.Unix(time.Now().Unix(), 0).UTC()
+	testTime := time.Unix(time.Now().Unix(), 0)
```

Comment:
Why does it fail? The code should not be sensitive to that and all operations should use UTC timestamps

Reply:
I changed the serialization logic so it uses UTC now - it seems there was an issue with unmarshalling `metav1.Time` and it would convert those timestamps to the local timezone instead